tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -604,3 +606,45 @@ def test_positive_sync_overview_widget(session):
         assert sync_values[0]['Product'] == product_name
         assert sync_values[0]['Status'] == 'Syncing Complete.'
         assert 'ago' in sync_values[0]['Finished']
+
+
+@tier2
+def test_positive_virtwho_configs_widget(session):
+    """Check if Virt-who Configurations Status Widget is working in the Dashboard UI
+
+    :id: 6bed6d55-2bd5-4438-9f72-d48e78566789
+
+    :Steps:
+
+        1. Create a Virt-who Configuration
+        2. Navigate Monitor -> Dashboard
+        3. Review the Virt-who Configurations Status widget
+
+    :expectedresults: The widget is updated with all details.
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    entities.VirtWhoConfig(
+        name="example_config{}".format(gen_integer()),
+        organization=org,
+        hypervisor_server=settings.clients.provisioning_server,
+        satellite_url=settings.server.hostname,
+        hypervisor_type='libvirt',
+        hypervisor_username='root',
+        hypervisor_id='hostname',
+        hypervisor_password='',
+    ).create()
+
+    with session:
+        session.organization.select(org_name=org.name)
+        values = session.dashboard.read('VirtWhoConfigStatus')
+        assert values['config_status'][0]['Configuration Status'] == 'No Reports'
```

Comment:
At least optimize in that way. Maybe it can be even better way
```
expected_values = [
    {'Configuration Status': 'No Reports', 'Count': 1}, 
    {'Configuration Status': 'No Change', 'Count': 0},
    {'Configuration Status': 'OK', 'Count': 0},
    {'Configuration Status': 'Total Configurations', 'Count': 1}
]
assert values['config_status'] == expected_values
```

Reply:
@oshtaier done , please revisit .